comeet/Data/Extensions/Date+API.swift

```diff
@@ -12,24 +12,28 @@ extension Date {
     
     static func fromAPI(stringDate: String) -> Date? {
         let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss-SSSS"
```

Comment:
Those last 4 "-SSSS" are actually a timezone offset.  It could also be "+SSSS" in the Eastern hemisphere.  Does iOS not have an ISO 8601 formatter?  I know it sucks that the API isn't super consistent in reprocessing dates in data fields into that standard, but It's supposed to use 8601 exclusively in url and form parameters.